app/views/jobseekers/profiles/personal_details/name.html.slim

```diff
@@ -2,7 +2,6 @@
 
 - content_for :breadcrumbs do
   nav.govuk-breadcrumbs aria-label="Breadcrumbs"
-  = govuk_back_link text: t("buttons.back"), href: back_url, html_attributes: { "aria-label" => "Back navigation", role: "navigation" }
```

Comment:
I believe that the error was raised by `back_url`, a helper method which is defined [here](https://github.com/DFE-Digital/teaching-vacancies/blob/2450de3af7e6d4767cb1b804a1ef8ad278a5a0a3/lib/multistep/controller.rb#L74). This method in turn calls the `previous_step` method.